README.md

```diff
@@ -18,4 +18,18 @@ A simple run would be the following:
 * Replace `<auxiliary-rpc>` with auxiliary chain's rpc url.
 * Replace `<db-path>` with the database path. It is the path for `sqlite` database. If not provided,it would create it.
 * `--force` option is used to forcefully override facilitator config. It is optional parameter.
-	
\ No newline at end of file
+	
+## Facilitator Start
```

Comment:
1. You are correct that in case of origin it's the name of the origin chain and for auxiliary it's the id.
2. `--mosaic-config` and `facilitator-config` are optional for second case.

Reply:
It's still not clear to me why `--mosaic-config` and `--facilitator-config` options are available for the second case. Please, document that part. Let reader to know what happens when it's optional, why one would like to provide that options, etc.

Reply:
I have updated the readme.